grafana-plugin/src/containers/EscalationVariants/EscalationVariants.tsx

```diff
@@ -125,6 +127,7 @@ const EscalationVariants = observer(
             </>
           )}
           <div className={cx('assign-responders-button')}>
+            {withLabels && <Label>Assign additional responders by user or by schedule</Label>}
             <WithPermissionControlTooltip userAction={UserActions.AlertGroupsWrite}>
               <Button
                 icon="users-alt"
```

Comment:
Few lines below this comment there is a "Add responders" button text. I think since we're updateing logics and those responders are optional, worth changing the text to "Add additional responders"